### PR TITLE
Release @google-cloud/spanner v2.1.0

### PR DESCRIPTION
**Pull request by callmehiphop**
This pull request was generated using releasetool.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,92 +2,62 @@
 
 [npm history][1]
 
-[1]: https://www.npmjs.com/package/nodejs-spanner?activeTab=versions
```
Comment (JustinBeckwith) [inline CHANGELOG.md]: This doesn't look right *scratches head* it looks like the v2 changelog is getting written over.  No idea how that happened :(

Comment (callmehiphop) [reply CHANGELOG.md]: oh no, should it be additive?

Comment (JustinBeckwith) [reply CHANGELOG.md]: Yup.  Idea is to have an entry in here for every release.

Comment (JustinBeckwith) [reply CHANGELOG.md]: To be clear, the tool should do this auto-magically.  I have no idea how you got in this state :/

Comment (callmehiphop) [reply CHANGELOG.md]: Thats my mistake, when the PR was made it looked to be based off of my forks master state (which is super out of date) and I rebased so I wouldn't have to re-sort the changelog.

**Review by JustinBeckwith — APPROVED** [review]

**Comment by callmehiphop** [comment]
Release is at https://github.com/googleapis/nodejs-spanner/releases/tag/v2.1.0

**Pull request merged**